packages/light-electron/package.json

```diff
@@ -28,7 +28,7 @@
     "channel": "beta"
   },
   "scripts": {
-    "prebuild": "cp -r ../light-react/build static/build && ./scripts/fixElectronBug.sh",
+    "prebuild": "rimraf static/build && cp -r ../light-react/build static/build && ./scripts/fixElectronBug.sh",
```

Comment:
Look at https://www.npmjs.com/package/copyfiles at some point (no need to change, just log/make a note - at some point probably want this to work on Windows as well)

Reply:
Aha, nice! Actually i will change it now so that it's done, or else I'll forget it